_includes/graphql/objects.md

```diff
@@ -238,3 +238,95 @@ mutation aNestedMutation {
   }
 }
 ```
+
+## Cloud Code Resolvers
+
+If you need more control over how your mutations modify data than what Parse's auto-generated mutations can provide, Cloud Code functions can be used as custom resolvers.
+
+For example, if you have classes named `Item` and `CartItem` in the schema, you can create an `addToCart` custom mutation that tests whether a specific item is already in the user's cart. If found, the cart item's quantity is incremented by one. If not, a new `CartItem` object is created.
+
+The ability to branch your resolver logic in this way enables you to replicate functionality found in Parse's auto-generated `createCartItem` and `updateCartItem` mutations and combine those behaviors into a single custom resolver.
+
+```sh
+# schema.graphql
+extend type Mutation {
+  addToCart(id: ID!): CartItem! @resolve
+}
+```
+
```

Comment:
We need to show how to pass the schema into the ParseGraphQL Server, here developers can think that a simple `schema.graphql` at the root of the folder can do the job

Reply:
Agreed. I could use the example setup as found in [README.md](https://github.com/parse-community/parse-server#customizing-your-graphql-schema). I'm hesitant to include the custom schema setup code here, as it seems to be a related topic, and not quite this same one. Would it be okay for me to include the custom schema setup in the `Getting Started` section instead? If so, it could be inserted in one of a couple places:
1. In the `Using Express.Js` sub-section (mixed in with the other ParseServer setup code), or
2. Given its own sub-section named something like, `Adding Custom Schema` (I think I would prefer this method)